src-tauri/src/state/reducers/credential_offer.rs

```diff
@@ -258,6 +340,12 @@ pub async fn send_credential_request(state: &mut AppState, action: Action) -> Re
 
         info!("generated hash-key: {:?}", key);
 
+        persist_asset(
+            "6440ceac338a920197100e60_NGDIL%20Logo%20Dark.svg",
+            key.to_string().as_str(),
+        )
+        .ok();
```

Comment:
What does this do? What is the significance of this: `"6440ceac338a920197100e60_NGDIL%20Logo%20Dark.svg"`?